src/main/java/com/example/osk/service/ClientService.java

```diff
@@ -0,0 +1,40 @@
+package com.example.osk.service;
+
+import com.example.osk.model.Client;
+import com.example.osk.repository.ClientRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class ClientService {
+
+    private final ClientRepository clientRepository;
+
+    @Autowired
```

Comment:
tutaj nie jest to potrzebne, ale nie jestem pewien czy powinno się to dodawać - taki temat do rozmyślenia :)